Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3_test.py

```diff
@@ -1783,3 +1783,75 @@ def test_create_indicators_list(names_and_indicators_list, notes, expected_resul
     """
     result = create_indicators_list(names_and_indicators_list, notes)
     assert result == expected_result
+
+
+@pytest.mark.parametrize(
+    "args, expected_data, expected_output",
+    [
+        (
+            {"indicator_ids": "123,456", "tags": "tag1,tag2"},
+            {
+                "ids": ["123", "456"],
+                "tags": [{"name": "tag1", "tlp": "red"}, {"name": "tag2", "tlp": "red"}],
+            },
+            "The tags have been successfully added for the following ids:\n `123, 456`",
+        )
+    ],
+)
+def test_add_indicator_tag_success(
+    mocker, args: dict[str, str], expected_data: dict[str, str], expected_output: str
+):
+    """
+    Given:
+        - A list of indicator IDs and tags
+    When:
+        - `add_indicator_tag_command` is called with valid arguments
+    Then:
+        - A POST request should be made to add the tags
+          and it should return a successful CommandResults
+    """
+    # Arrange
+    client = mock_client()
```

Comment:
instead of `client = mock_client()`, add `mock_client` to the input args. Pytest will automatically add it.